### PR TITLE
[IMP] Discuss Team Comms Add Chatter Info

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: https://www.odoo.com/web#id=3464452&cids=3&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@Felicious @jevs-odoo  This small change on the team communication doc is ready for your review when you have a moment.

Thanks :+1: 

Tim

**Comment by tiku-odoo** [comment]
@StraubCreative 

This change is priority for the Discuss app-- ready for your merge when you have a moment. 

Thanks, Tim

File: content/applications/productivity/discuss/overview/team_communication.rst [hunk 1]
```diff
@@ -97,3 +97,18 @@ drop-down menu.
...
    - :doc:`get_started`
    - :doc:`plan_activities`
+
+Linking channel in chatter
+==========================
+
+Channels can be linked in the chatter (log note) of a record in Odoo. To do so, simply type: `#` and
+the channel name. Click or press enter on the *channel* name. Upon logging the note a link to the
+channel will appear. After clicking on the link a chat window with the channel conversation will
+pop-up in the lower right corner of the screen.
```
Comment (StraubCreative) [inline content/applications/productivity/discuss/overview/team_communication.rst:107]: **pop-up** is a compound adjective
**pop up** is the verb you're looking for ;)
```suggestion
pop up in the lower right corner of the screen.
```

**Review by StraubCreative — APPROVED** [review]
Look good @tiku-odoo 
Merge when ready 👍 

@robodoo delegate=tiku-odoo

**Comment by tiku-odoo** [comment]
@StraubCreative 

Pushed up requested change and put "other doc links" at the bottom of the doc (see also). I did this to keep in line with the other overview docs, which are linked at the bottom of all the docs.  

Ready to merge.

**Comment by tiku-odoo** [comment]
@robodoo r+

**Comment by tiku-odoo** [comment]
@robodoo retry

**Pull request closed without merging**